src/main/java/uk/sky/cqlmigrate/CassandraLockingMechanism.java

```diff
@@ -127,6 +127,7 @@ public boolean release(String clientId) throws CannotReleaseLockException {
             }
 
         } catch (WriteTimeoutException e) {
+            log.info("Lock release timed out for client {}", clientId);
```

Comment:
I think we should make that warning rather than info. A similar case can be found in the **acquire** method